src/Raven.Server/Documents/Sharding/Handlers/ShardedIncomingReplicationHandler.cs

```diff
@@ -49,6 +47,15 @@ public class ShardedIncomingReplicationHandler : AbstractIncomingReplicationHand
             }
         }
 
+        public StreamsTempFile GetTempFile()
+        {
+            DevelopmentHelper.ShardingToDo(DevelopmentHelper.TeamMember.Shiran, DevelopmentHelper.Severity.Normal, "figure out if the sharded db is encrypted ('encrypted' parameter for 'StreamsTempFile')");
```

Comment:
you are already passing `_parent.Context.Encrypted` there. What else needs to be done?

Reply:
Nothing :-) done